generic/tests/pktgen_perf.py

```diff
@@ -16,7 +16,7 @@
 _system_output = functools.partial(process.system_output, shell=True)
 
 
-def format_result(result, base="33", fbase="2"):
+def format_result(result, base="12", fbase="2"):
```

Comment:
IIRC this was changed before, do we consider make it configurable in cfg file?

Reply:
I have udpated, and passed tests. 


JOB LOG    : /root/avocado/job-results/job-2022-01-07T04.45-1bae5d7/job.log|
(1/5) 82599.bridge_test.1q.acceptance.repeat1.Host_RHEL.m8.u6.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.pktgen_perf.q35:  PASS (221.01 s)
 (3/5) 82599.bridge_test.1q.acceptance.repeat3.Host_RHEL.m8.u6.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.pktgen_perf.q35: PASS (221.79 s)
 (4/5) 82599.bridge_test.1q.acceptance.repeat4.Host_RHEL.m8.u6.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.pktgen_perf.q35: PASS (223.50 s)
 (5/5) 82599.bridge_test.1q.acceptance.repeat5.Host_RHEL.m8.u6.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.6.0.x86_64.io-github-autotest-qemu.pktgen_perf.q35: PASS (220.69 s)
RESULTS    : PASS 5 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2022-01-07T04.45-1bae5d7/results.html
JOB TIME   : 1154.75 s